public/locales/en/translation.json

```diff
@@ -1,427 +1,427 @@
-    "where_to_find": "Confession times are listed in your local parish bulletin, and you can find them online at your parish website or at <mass>masstimes.org</mass>. You can also schedule a confession at any time you'd like by contacting your local parish.",
-    "about_confession_end": "When you go to confession, typically, you will have the choice of kneeling anonymously behind a screen or sitting face-to-face with your confessor.Don't be nervous about going to confession! Whatever you confess, your priest has heard it before. Remember, he is there to help you.",
-    "about_this_app": "About This App",
-    "this_app_is_designed": "This app is designed to help Roman Catholics prepare for the sacrament of confession by examining their conscience. It is <strong>NOT</strong> a substitute for confession.",
-    "please_be_respectful": "Please be respectful of those around you when you use this app. I recommend that you turn your phone off when you're inside your church, and use this app before you arrive. If you do use this app inside your church or during confession, please ensure your phone is in silent mode.",
-    "this_website": "This website, <website>ConfessIt.app</website>, is based on the <app>ConfessIt Android App</app> (created in 2012 by the same developer). While it's not (yet) a complete reproduction, it aims to make the app available to a wider range of users on a broader range of devices. (This site works on iOS, Android, tablets, and computers!)",
-    "if_you_find": "If you find this app useful, please consider <strong>sharing it</strong> with your friends and family. Tell people about it on Facebook, Reddit, Twitter, at your church, or in your Bible study group to help spread the word!",
-    "privacy": "Privacy",
-    "information_you_enter": "Information you enter into this app is only stored on your device. It is not sent over the internet. We are able to do this using a technology provided by your web browser called <a>local stroage</a>. We do not run Google Analytics or any other data collection mechanism on this site. Data you enter will be saved on your device until you hit <kbd>Clear</kbd> even if you close the window or refresh the page.",
-    "open_source": "Open Source",
-    "confessit_is_open_source": "ConfessIt is open source. We develop the app on <i></i> <github>GitHub</github> and we collaborate in the <osc>Open Source Catholic</osc> community on Slack.",
```

Comment:
@JohnRDOrazio This `<i></i>` was correct. It represents the GitHub fontawesome icon in our React i18n string - see the rendered version near the bottom of https://confessit.app/about

Reply:
oh my, I would have expected to see something like `<i className="fab fa-github"></i>`. I'll add a note in weblate for all translators explaining what this is. Just out of curiosity, how does that then become the github icon?

Reply:
should a github icon appear here too? https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=cf7c69f51e27cb84

Reply:
I see now where it's being transformed:

https://github.com/kas-catholic/confessit-web/blob/c987b52e39adc99fc26b93bcbb26ae7572825999/src/About.js#L204-L214